.changeset/ninety-otters-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-lighthouse-backend': minor
+---
+
+Updated Lighthouse schedule configuration to fix crashes when using custom configuration
```

Comment:
may want to mention in here that the config format has changed, and that you may want to refactor your config from X to Y - existing users will appreciate the info